Documentation/ceph-csi-troubleshooting.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Ceph-CSI Configuration
+weight: 1200
+indent: true
+---
+
+# Troubleshooting Ceph-CSI
+
+The issue in Provisioning the CephFS/RBD PVC or Mounting the CephFS/RBD PVC to application pods can happen for Many reasons like Network connectivity between CSI pods and ceph, cluster health issue, Slow operations, kubernetes issue, etc. Even the issue can be at the Ceph-CSI layer itself. Hope the below-documented steps might help you to understand and Debug the ceph-csi issues.
+How to identify network issues
+
+Get the monitor IP from the configmap you have created when starting the ceph-csi pods or from the ceph cluster.
+
+```console
+# ceph mon dump
+dumped monmap epoch 1
+epoch 1
+fsid ba41ac93-3b55-4f32-9e06-d3d8c6ff7334
+last_changed 2021-01-20T12:28:37.972517+0000
+created 2021-01-20T12:28:37.972517+0000
+min_mon_release 15 (octopus)
+0: [v2:10.111.136.166:3300/0,v1:10.111.136.166:6789/0] mon.a
+```
+
+
+**Note** :- `10.111.136.166` is the Monitor IP and `6789` is the monitor port that will be used by Ceph-CSI to connect to the ceph cluster.
+
+**Ceph Monitors normally listen on port 3300 for the new v2 protocol, and 6789 for the old v1 protocol. Check if your cluster is listening on v2 or v1 protocol.**
```

Comment:
Instead of bold, this should indent it:
```suggestion
> Ceph Monitors normally listen on port 3300 for the new v2 protocol, and 6789 for the old v1 protocol. Check if your cluster is listening on v2 or v1 protocol.
```